packages/mobile/src/fiatExchanges/saga.ts

```diff
@@ -91,10 +100,40 @@ function* bidaliPaymentRequest({
   }
 }
 
+export function* searchNewItemsForProviderTxs({ transactions }: NewTransactionsInFeedAction) {
+  try {
+    if (!transactions || !transactions.length) {
+      return
+    }
+    Logger.debug(TAG + 'searchNewItemsForProviderTxs', `Checking ${transactions.length} txs`)
+
+    const providerAddresses = yield select(providerAddressesSelector)
+
+    for (const tx of transactions) {
+      if (tx.__typename !== 'TokenTransfer' || tx.type !== TokenTransactionType.Received) {
+        continue
+      }
+
+      if (providerAddresses.indexOf(tx.address) >= 0) {
```

Comment:
Nit: can we use `includes` instead of `indexOf`? Effectively the same thing but just seems a little more appropriate

Reply:
I don't know why I thought this didn't exist. It has for quite a while now, I've been using this less readable way for a long time for no reason :/